### PR TITLE
move port label to api repo

### PR DESCRIPTION
**Pull request by xichengliudui**
ref: https://github.com/istio/istio/pull/36592

### CODE REVIEW

File: annotation/annotations.yaml [hunk 1]
```diff
@@ -61,6 +61,15 @@ annotations:
...
       - Service
 
+  - name: networking.istio.io/gatewayPort
+    featureStatus: Alpha
+    description: IstioGatewayPortLabel overrides the default 15443 value to use for a multi-network gateway's port
+    deprecated: false
+    hidden: false
+    resources:
+      - Service
+      - Pod
```
Comment (howardjohn) [inline annotation/annotations.yaml:71]:
```suggestion
```

Only on service

**Review by howardjohn — APPROVED** [review]

File: annotation/annotations.yaml (hunk 1 above)
Comment (xichengliudui) [reply annotation/annotations.yaml]: done

**Review by howardjohn — APPROVED** [review]

**Review by ericvn — APPROVED** [review]
LGTM. Will need a release note either here or in the istio/istio PR.

**Pull request merged**